src/cli/pack_command.zig

```diff
@@ -2380,6 +2445,32 @@ pub const PackCommand = struct {
 
         Output.flush();
     }
+
+    /// Some files are always packed, even if they are explicitly ignored or not
+    /// included in package.json "files".
+    fn isUnconditionallyIncludedFile(filename: []const u8) bool {
+        return filename.len > 5 and (stringsEql(filename, "package.json") or
+            isSpecialFileOrVariant(filename, "LICENSE") or
```

Comment:
This will change the behavior, allowing `LICENSE.foo`. We should check if npm does the same, and if not change to so it only matches `LICENSE` exactly.